OmiseSDK/OmiseJsonParser.swift

```diff
@@ -1,6 +1,6 @@
 import Foundation
 
-@objc(OMSOmiseJsonParser) public final class OmiseJsonParser: NSObject {
```

Comment:
that's nice. but i think it'd make confuse our users. better have a single name for it.

Let's do a rename sweep in vNext